app/views/posts/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1>HERE ARE MY POSTS</h1>
+
+<div class="container">
+  <div class="details">
+    <div class="user-card">
+      <img src="<%= @user.photo %>">
+      <div class="user-card-plate">
+        <%= @user.name %>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </div>
+  <% @posts.each do |post| %>
+    <div class="post-card">
+      <div class="post-number">
+        <h2><%= link_to post.title, user_post_path(user_id: @user.id, id: post.id) %></h2>
+      </div>
+      <div class="post">
+        <p><%= post.text %></p>
+      </div>
+      <div class="comment">
+        <p>Number of comments: <%= post.comments_counter %></p>
+        <p>Number of likes: <%= post.likes_counter %></p>
+      </div>
+      <div class="my-comments">
+        <% post.comments.each do |comment| %>
+          <div><%= post.user.name %> <%= comment.text %></div>
+        <% end %>
+      </div>
+    </div>
```

Comment:
- Good job done so far, but according to the project requirements, you are required to use the `post.recent_comments` in the Post model to render the 5 most recent comments. Please fix this 👍 👍